### DISCUSSION TITLE
3차 데모데이 끝나고 이야기할게요

### DISCUSSION THREAD

**Discussion by kyum-q** [woowacourse-teams/2024-code-zap#388] [이야기해봐요]
> 오늘 4시 50분 회의

### 새로운 결정 사항이 있을 때
- 의견 발제자가 디스커션 및 이슈 발행
   - 디스커션에 자신의 주장을 적기
   - 안내하기

>  회의 공지 스레드에 공유하기

### 앞으로 회의가 필요할 때 
- 위 의견에 동의하지 않을 경우, 댓글로 반박 후 회의 진행
- 발제자가 회의 진행

### 발제자가 없을 경우
- 뽑기(마위가 가져옴)

## 이번 스프린트에 대한 회고 (0분)

- Z 스프린트 끝나는 날 6시에 사용자에게 배포

## 다음 스프린트 git 전략 수정 (10분) -> 다수결 결정

**Comment by kyum-q** [comment]
# 데모데이 피드백 검토 (20분)

**Reply by kyum-q** [reply]
- 위키 디스커션 링크를 모두 넣어주자

**Reply by kyum-q** [reply]
- 에러로깅 관련 논의 추가로 필요하다

**Reply by zangsu** [reply]
헤인 : 에러 코드를 같이 정하는 논의가 필요하다 -> 언제 논의 할 것인지 정하고, 같이 의견을 미리 작성해 봐요

**Reply by kyum-q** [reply]
**랜딩 페이지**
- 홈화면이 처음에 나왔으면 좋겠다.
- 코드잽의 타겟에 대한 설명, 왜 그런 분에게 우리 서비스가 필요한가??

**Reply by kyum-q** [reply]
- 의사결정을 할 땐, 문서화해라! 우리 팀에 누군가 합류했을 때 근거로 보여줄 수 있도록!

**Reply by kyum-q** [reply]
- 회원가입 페이지가 우리 서비스의 사용자가 처음 이용하는 기능인데, 생각보다 불친절했다. UX를 친절하게, 그리팅을 잘해주자

**Reply by kyum-q** [reply]
### 새로운 결정 사항이 있을 때
- 의견 발제자가 디스커션 및 이슈 발행
   - 디스커션에 자신의 주장을 적기
   - 안내하기

### 앞으로 회의가 필요할 때 
- 위 의견에 동의하지 않을 경우, 댓글로 반박 후 회의 진행
- 발제자가 회의 진행

### 발제자가 없을 경우
- 뽑기(마위가 가져옴)

**Comment by kyum-q** [comment]
# 이번 스프린트에 대한 회고 (20분)

**Comment by kyum-q** [comment]
# 다음 스프린트 git 브랜치 전략 수정 (10분) -> 다수결 결정

### A안
- main
  - dev
    - dev/be
    - dev/fe

### B안
- main
  - dev/be
  - dev/fe

**Reply by kyum-q** [reply]
#390
